<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SetDatabase</class>
 <widget class="QDialog" name="SetDatabase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>486</width>
    <height>312</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>数据库</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>icon/dns-black-48dp.svg</normaloff>icon/dns-black-48dp.svg</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="0">
    <widget class="QPushButton" name="pushButtonTestConnection">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>测试</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="3">
    <widget class="QPlainTextEdit" name="plainTextEditTestResult">
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="plainText">
      <string>准备就绪</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="0" colspan="3">
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="labelHost">
       <property name="text">
        <string>地址</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEditHost">
       <property name="text">
        <string>127.0.0.1</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="labelPort">
       <property name="text">
        <string>端口</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="lineEditPort">
       <property name="inputMethodHints">
        <set>Qt::ImhFormattedNumbersOnly</set>
       </property>
       <property name="text">
        <string>3306</string>
       </property>
       <property name="maxLength">
        <number>5</number>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="labelUser">
       <property name="text">
        <string>用户名</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="lineEditUser">
       <property name="text">
        <string>root</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="labelPassword">
       <property name="text">
        <string>密码</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="lineEditPassword">
       <property name="echoMode">
        <enum>QLineEdit::PasswordEchoOnEdit</enum>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="labelDatabase">
       <property name="text">
        <string>数据库</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="lineEditDatabase">
       <property name="text">
        <string>passwords</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SetDatabase</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>398</x>
     <y>152</y>
    </hint>
    <hint type="destinationlabel">
     <x>242</x>
     <y>122</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SetDatabase</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>398</x>
     <y>152</y>
    </hint>
    <hint type="destinationlabel">
     <x>242</x>
     <y>122</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
